pkg/runutil/runutil.go

```diff
@@ -70,6 +70,24 @@ func Repeat(interval time.Duration, stopc <-chan struct{}, f func() error) error
 	}
 }
 
+// Forever executes f every interval seconds until stopc is closed. Errors are logged and ignored.
+// It executes f once right after being called.
+func Forever(logger log.Logger, interval time.Duration, stopc <-chan struct{}, f func() error) {
```

Comment:
Do we really need this? Why not `Repeat` with `func()` never returning error? (:

Reply:
I would remove it IMO unless you have strong opinion on this (:

Reply:
Fixed at f4ff6df2684d46baa4585f6cfd0115e6780bbb63